includes/class-sensei-course.php

```diff
@@ -3886,6 +3889,28 @@ public function has_sensei_blocks( $course = null ) {
 
 		return false;
 	}
+
+	/**
+	 * Alters the redirect url after a user enrols to a course.
+	 *
+	 * @param String  $url  Default redirect url.
+	 * @param WP_Post $post Post object for course.
+	 *
+	 * @return String
+	 */
+	public static function alter_redirect_url_after_enrolment( $url, $post ) {
+
+		$course_id = $post->ID;
+		if ( Sensei_Course_Theme::has_sensei_theme_enabled( $course_id ) ) {
```

Comment:
You probably could use the existing `should_use_sensei_theme_template` method from the same class here.

BTW, probably the name you used is much better now. Maybe we could rename that method too (Normally we'd deprecate it because we already released this method, but since it's something "experimental/in development", probably it should be okay to just rename it).

Reply:
@renatho sorry for the late reply. I had to modify that method you are referring to a little for my use.
Because  `should_use_sensei_theme_template` also checks that the current post type is a lesson or quiz, but I wanted only to check if the given course ID had the theme enabled.
See my changes at line 118 in `class-sense-course-theme.php`